packages/navitem/src/react/common.tsx

```diff
@@ -0,0 +1,59 @@
+import Halo from '@pluralsight/ps-design-system-halo'
+import { ValueOf, stylesFor } from '@pluralsight/ps-design-system-util'
+import { compose, css } from 'glamor'
+import React, { useContext } from 'react'
+
+import stylesheet from '../css'
+import { alignments } from '../vars'
+
+import Context from './context'
+
+const styles = {
+  bar: (props: {
+    alignment: ValueOf<typeof alignments>
+    selected: boolean
+    // eslint-disable-next-line camelcase
+    UNSAFE_stylesFor: unknown
+  }) => {
+    const isHorz = (props.alignment = alignments.horizontal)
+    const isVert = (props.alignment = alignments.vertical)
```

Comment:
These look fishy to me. Seems like this should be a test (==) instead of assignment (=).

Reply:
ha. good catch 🤦